test_crates/trait_unsafe_removed/old/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+// Public unsafe trait becomes safe, should get reported.
+pub unsafe trait TraitBecomesSafe {}
+
+// Trait that was not publicly-visible becomes safe, shouldn't get reported.
+unsafe trait TraitBecomesPublicAndSafe {}
```

Comment:
Consider the reverse test case here as well: `pub unsafe trait` becomes safe and private.